dna/clutter/clutter.js

```diff
@@ -33,7 +33,7 @@ function unfollow(userAddress){
     var me = getMe();
     return commit("unfollow",  // On my source chain, commits the unfollow entry
                   {Links:[
-                      {Base:userAddress,Link:me,Tag:"follower",LinkAction:HC.LinkAction.Del},
+                      {Base:userAddress,Link:me,Tag:"followers",LinkAction:HC.LinkAction.Del},
```

Comment:
So I don't really understand Links yet, but I'd like to. 
For a given situation where one user follows another there are two entries, or this is one entry?
One from the otherUserAddress to the "meAddress" saying include "me" in their followers.
One from the meAddress to the otherUserAddress saying include them in my list of following.

Why (how?) are both written into this entry, in this array?

Reply:
A links entry that you commit to your chain is a list of links.  It has the side effect of adding the Link+Tag to the base hash on the DHT for each of the links in the list.  Thus when you do a getLinks(base,tag) it will return all the links of that tag on that base.  This is how we do relations in a sparse hash space.   

Every "commit" is something that you say/declare, so it get's written to your chain.  What you say is also shared to the DHT.  Most things are shared simply, i.e. the entry is shared and some subset of nodes holds the entry for access via get().  Links are a bit more complicated in that one links commit results in multiple shares to the DHT, one per link.

Reply:
Oooooh it's like (vaguely) an RDF triple. I get it. The "tag" is specifying what type of relation?

Reply:
Yah but it's not subject/object/predicate.  You can use the tag as any kind of semantic marker, and actually sometimes it might even be useful to put actual data into the tag.  I'm guessing people are going to come with all kinds of fun things.  It basically allows us to convert a hash data store into a graph.